File: compiled_lang/go/src/j10.arguments.go
package main

import (
	"fmt"
	"os"
)

func main() {
	args := os.Args[1:]

	fmt.Println("The arguments passed are:")
	for i, arg := range args {
		fmt.Printf(" item %d: %s\n", i+1, arg)
	}
}

File: lessons/compiled_lang/cpp/src/j10.arguments.cpp
#include <iostream>

int main(int argc, char* argv[]) {
    std::cout << "The arguments passed are:" << std::endl;
    for (int i = 1; i < argc; i++) {
        std::cout << " item " << i << ": " << argv[i] << std::endl;
    }

    return 0;
}
